config/gateway-api/crd/standard/gateway.networking.k8s.io_gatewayclasses.yaml

```diff
@@ -234,6 +234,9 @@ spec:
         - jsonPath: .metadata.creationTimestamp
           name: Age
           type: date
+        - jsonPath: .status.supportedFeatures
+          name: SupportedFeatures
+          type: string
```

Comment:
Is this change from the upstream gep too? 
Is it needed for the metrics piece?
Asking as I'm not sure what the impact is i.e. having the output be a string format when it's actually an  array of strings. Maybe that's ok?

Reply:
Oh yes forgot about this, it is not needed.